workers/data_refinery_workers/processors/utils.py

```diff
@@ -49,13 +49,14 @@ def end_job(job_context: Dict):
     job.end_time = timezone.now()
     job.save()
 
+    batches = job_context["batches"] if "batches" in job_context else None
+
     # Clean up temp directory.
-    if len(job_context["batches"]) != 0:
+    if batches is not None and len(batches) > 0 and len(batches[0].files) > 0:
```

Comment:
Not checking if batches[0] has a files key

Reply:
Do you think that's really necessary? This is a list of `Batch` objects which each have a `files` attribute, it's not looking up a key.